src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -172,7 +172,7 @@ def handler(ex):
             t_error = get_sdk(self.command_loader.cli_ctx,
                               ResourceType.DATA_STORAGE,
                               'common._error#AzureHttpError')
-            if isinstance(ex, t_error) and ex.status_code == 403:
+            if isinstance(ex, t_error) and ex.status_code == 403 and ex.error_code == 'AuthorizationPermissionMismatch':
```

Comment:
>AuthorizationPermissionMismatch [](start = 87, length = 31)

How do you know which error_code should use? Is there a document about the error codes and their meaning?